draft-ietf-rats-corim.md

```diff
@@ -2417,6 +2499,35 @@ If key verification succeeds for any _key_:
 
 Otherwise, do not add the `addition` ECT to the ACS.
 
+#### Processing Domain Membership {#sec-process-dm}
+
+This section assumes that each Domain Membership Triples has been transformed into an internal representation following the steps described in {{sec-ir-dm-trans}}, resulting in the representation specified in {{sec-ir-dm}}.
+
+Domain Membership ECTs (`cmtype`: `domain-member`) are matched with ACS entries (of `cmtype`: `evidence`) using the following algorithm:
+
+* For every `domain` entry:
+  * For each `i` in `members`, check that there is an ACS entry with a matching `environment` and cm-type = `evidence`
+  * If all `members` match an a ACS entry, add the `domain` ECT to ACS
+
+* If there is a partial match between the `member` environments and the ACS ECT `environment`, three separate cases must be considered.
+
+  ACS ECT contains `N` environments while Domain ECT `members` reports `M` Environments:
+
+  1. `N` >= `M` and some entries of `M` match.
+  This is a case of a Composite Attester, where other entries of `M`, may itself be domain identifiers for other `domains`.
```

Comment:
M is a natural number. What does it mean to be an entry of M? I think we need to say the DM-ECT members.

There's a plurality mismatch here too: "entries"/"itself".

Reply:
New text refers to `i` in `members` which is much cleaner way of representation!